### PR TITLE
add supplier types

### PR DESCRIPTION
**Pull request by cactusaurelius**
## :memo: Description

Knawat supplier types

- [x] New feature (non-breaking change which adds functionality)

## :checkered_flag: Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: src/supplier.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+export interface Supplier {
+    id: string;
```
Comment (HazemKhaled) [inline src/supplier.ts:3]: make sure to have all fields from here https://docs.knawat.io/?url=https://suppliers.knawat.io/openapi/openapi.json#tag/Suppliers/paths/~1suppliers~1{id}/get

File: src/supplier.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    integrations: {
+      type: 'ideasoft';
+      data: {
+        access_token: string;
+        fields: any;
+      };
+    };
+  }
+  
+  export interface Webhook {
```
Comment (HazemKhaled) [inline src/supplier.ts:20]: create file for each type, this should be in webhook.ts
```json
{
  "id": "5f3bc66f576f0d3768be3376",
  "topic": "porders.create",
  "supplierId": "ID",
  "url": "https://example.com/webhook",
  "createdAt": "2020-08-18T14:55:42.941+03:00",
  "updatedAt": "2020-08-18T14:55:42.941+03:00"
}
```

**Review by HazemKhaled — CHANGES_REQUESTED** [review]

**Comment by HazemKhaled** [comment]
@neer14 once approved request my review please

File: src/supplier.ts [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  url: string;
+  integrations: {
+    enabled: boolean;
+    type: 'ideasoft';
+    data: {
+      access_token: string;
+      fields: any;
+    };
+  };
+}
```
Comment (neer14) [inline src/supplier.ts:58]: missing jscrapper_urls
last_translation_update should be Date

File: src/supplier.ts [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+interface TranslationPipe {
+  from: string;
+  to: string;
+  // TODO: enum apply_to
+  apply_to: string[]
```
Comment (neer14) [inline src/supplier.ts:64]: I will update appy_to later after pipeline pr is completed

File: src/supplier.ts (hunk 3 above)
Comment (cactusaurelius) [reply src/supplier.ts:59]: Done

**Pull request merged**